server/routes/notification.js

```diff
@@ -114,9 +116,10 @@ router.get(
       Promise.all([
         // This should be replaced with a non-paged call that shows only recent notifications.
         // This isn't an audit!!
-        notificationService.getShiftNotifications(req.user.username),
-        notificationService.getShiftNotificationsPaged(req.user.username, offset, perPage),
-      ]).then(([count, rows]) => {
+        DEPRECATEnotificationService.getShiftNotifications(req.user.username),
+        DEPRECATEnotificationService.getShiftNotificationsPaged(req.user.username, offset, perPage),
+        notificationService.getPreferences(req.user.token),
+      ]).then(([count, rows, preferences]) => {
```

Comment:
destruct out the param you need `]).then(([count, rows, { snoozeUntil }]) => {`

Reply:
I left this on the `preferences` object as I use the property name `snoozeUntil` to pass to ejs as the formatted copy. I could change the ejs key `snoozeUntil` to `formattedSnoozeUntil` if you prefer, this is probably easier to read and avoids my sneaky/bad poor naming

Reply:
You could also assign a new name in the deconstruct `]).then(([count, rows, { snoozeUntil: rawSnoozeUntil }]) => {`
I prefer externalising as a utility though. Its got a nice encapsulation about it?